WikipediaUnitTests/MWKImageFaceDetectionTests.m

```diff
@@ -96,7 +96,7 @@ - (void)testDetectingFacesInFacelessImage {
         .then(^(NSArray* faces) {
             [self.image setNormalizedFaceBoundsFromFeatures:faces inImage:facelessImage];
         });
-    } timeout:WMFDefaultExpectationTimeout WMFExpectFromHere];
+    } timeout:WMFDefaultExpectationTimeout * 2 WMFExpectFromHere];
```

Comment:
Ok - that Macro is… confusing… especially when it is sitting next to math… and being named like a constant.

We should look to do something to make it more idiomatic and clear in the future.